operations/app/src/modules/network/old-vnet.tf

```diff
@@ -0,0 +1,262 @@
+resource "azurerm_network_security_group" "nsg_public" {
```

Comment:
Everything in this file are existing resources that lives in `main.tf` prior to the PR. Names and configuration are identical, so Terraform recognizes these as the exact same resources with no modifications necessary to the state.

This was refactored so after we are done with the migration, we can delete this file, have Terraform remove all of the old resources, and all other files remain untouched.